src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -2029,6 +2030,46 @@ public void validateDeleteRequest(
     return rowCounts;
   }
 
+  private static final String exportToParquetTemplate =
+      "export data OPTIONS( "
+          + "uri='<exportPath><table>-*.parquet', "
+          + "format='PARQUET') "
```

Comment:
Should we add compression as an option too?

Reply:
For POC, I don't think we need compression. These are small datasets, so I don't think it buys us much. Its easy to add in the future, and I definitely think we'll support it, but it doesn't help the "get data into workspaces" effort right this very instant.